librato_python_web/instrumentor/web/django_.py

```diff
@@ -71,91 +72,79 @@ def process_exception(self, request, exception):
             telemetry.count('web.errors')
 
 
-def django_inject_middleware(original_method):
-    @wraps(original_method)
-    def decorator(*args, **keywords):
+def django_inject_middleware(original_method, *args, **keywords):
+    if not hasattr(django_inject_middleware, 'injected_middleware'):
+        # Do this the very first time only
         logger.info('injecting AgentMiddleware into django')
+        django_inject_middleware.injected_middleware = True
```

Comment:
Yep, I'd prefer for this to fail at most once, than to try repeatedly. Will wrap the following code in a try/except block.